### PR TITLE
Update documentation for monitoring

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/control-plane/metrics.md [hunk 1]
```diff
@@ -18,15 +18,35 @@ To enable monitoring system for PipeCD, you first need to set the following valu
...
 If you've already enabled monitoring system in the previous section, you can access Grafana using port forwarding:
 
 ```
 kubectl port-forward -n {NAMESPACE} svc/{PIPECD_RELEASE_NAME}-grafana 3000:80
 ```
 
-### Alert notifications
+#### Control-plane dashboards
+There are three dashboards related to Control-plane:
+- Overview - usage stats of PipeCD.
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/control-plane/metrics.md:30]: nit: remove the dot at end of the line

File: docs/content/en/docs/operator-manual/control-plane/metrics.md [hunk 2]
```diff
@@ -18,15 +18,35 @@ To enable monitoring system for PipeCD, you first need to set the following valu
...
-### Grafana dashboard
+## Dashboards
 If you've already enabled monitoring system in the previous section, you can access Grafana using port forwarding:
 
 ```
 kubectl port-forward -n {NAMESPACE} svc/{PIPECD_RELEASE_NAME}-grafana 3000:80
 ```
 
-### Alert notifications
+#### Control-plane dashboards
```
Comment (nghialv) [inline docs/content/en/docs/operator-manual/control-plane/metrics.md:28]: It would be nice if we could have some screenshots.

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/control-plane/metrics.md:28]: We're on the same page. I'm looking to add dashboard screenshots after finishing up it (there are a couple of things to improve actually)

File: docs/content/en/docs/operator-manual/control-plane/metrics.md (hunk 1 above)
Comment (nakabonne) [reply docs/content/en/docs/operator-manual/control-plane/metrics.md]: fixed 👍

**Pull request merged**